### DISCUSSION TITLE
How to use PHP config files in PhpStorm?

### DISCUSSION THREAD

**Discussion by stephanvierkant** [symfony/symfony#44216] [Q&A]
How do you use PHP config files in PhpStorm? Because they're in `var/cache/{env}` the config files are removed frequently and since I excluded `var/cache` in PhpStorm, I've to un-exclude them every time after a cache clear.

![Screenshot_20211123_122019](https://user-images.githubusercontent.com/601833/143015411-9fd04182-7435-45b2-86cd-f8593f5f0a13.png)

**Comment by stof** [comment]
@nicolas-grekas @jderusse maybe the generated code should be in a `var/generated` folder instead of `var/cache`, so that it is easier to exclude the cache folder from the IDE scanning without excluding the generated code.

**Comment by jderusse** [comment]
make sens to me
